easybuild/easyconfigs/t/Tk/Tk-8.6.8-GCCcore-6.4.0.eb

```diff
@@ -26,7 +26,10 @@ dependencies = [
 
 configopts = '--enable-threads --with-tcl=$EBROOTTCL/lib CFLAGS="-I$EBROOTTCL/include"'
 
-postinstallcmds = ["ln -s wish%(version_major_minor)s %(installdir)s/bin/wish"]
+postinstallcmds = [
+    "make install-private-headers",
```

Comment:
@hajgato Wouldn't it be better to do this via `installopts`?

```python
installopts = "&& make install-private-headers"
```

Also, we should enhance `sanity_check_paths` to check for at least one of these header files?

Reply:
@boegel Done.